app/assets/javascripts/darkswarm/controllers/group_page_controller.js.coffee

```diff
@@ -1,12 +1,10 @@
-Darkswarm.controller "GroupPageCtrl", ($scope, group_enterprises, Enterprises, MapConfiguration, OfnMap, visibleFilter, Navigation) ->
+Darkswarm.controller "GroupPageCtrl", ($scope, enterprises, Enterprises, MapConfiguration, OfnMap, visibleFilter, Navigation) ->
   $scope.Enterprises = Enterprises
 
-  all_enterprises_by_id = Enterprises.enterprises_by_id
+  enterprises_by_id = enterprises.map (enterprise) =>
+    Enterprises.enterprises_by_id[enterprise.id]
```

Comment:
Is `enterprises_by_id` a list or a map? The name indicates that it's a map but the code seems to indicate it's a simple list of enterprises. What is the goal here?